### PR TITLE
first-commit-3

### PR DESCRIPTION
**Pull request by yousef-najeh**

### CODE REVIEW

File: thirdstage-react/src/component/About/about.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import "./aboutStyle.scss"
+import Card from "../cards/AboutCard/aboutCards"
+const array=["7.jpg","8.jpg","9.jpg","7.jpg",]
+
+function About(){
+    return(
+        <div  id="About"
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/About/about.jsx]: remove extra space after div

File: thirdstage-react/src/component/About/about.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import "./aboutStyle.scss"
+import Card from "../cards/AboutCard/aboutCards"
+const array=["7.jpg","8.jpg","9.jpg","7.jpg",]
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/About/about.jsx]: it's better to import external packages at first then the internal ones 
and at last import css and images

Comment (amonamoqadi) [inline thirdstage-react/src/component/About/about.jsx]: add it inside About function

File: thirdstage-react/src/component/About/aboutStyle.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
+@use "../varibles.scss" as *;
+
+.about{
+    width:100%;
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/About/aboutStyle.scss]: add space

File: thirdstage-react/src/component/Contact/contact.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import Input from "../InputField/input"
+import "./contactStyle.scss"
+const PlaceHolders=["Name","Email","Subject","Comment"]
+
+function Contact(){
+
+    return(
+        <div id="Contact"
+             className="contact">
+            <p className="contact__header" >Contact</p>
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/Contact/contact.jsx]: remove extra space before >

File: thirdstage-react/src/component/Header/style.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        }
+}
+@media (max-width:$mediomSize){
+    .header{
+            height: 400px;
+    }
+}
+@media (max-width:$smallSize){
+    .header{
+            height: 200px;
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/Header/style.scss]: not formatted well

File: thirdstage-react/src/component/InputField/input.jsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import"./inputStyle.scss"
+
+function Input({placeHolder}){
+
+    return(
+        <input type="text"
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/InputField/input.jsx]: Do all inputs have type "text"?

File: thirdstage-react/src/component/InputField/input.jsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+    return(
+        <input type="text" 
+               placeholder={placeHolder}
+               className="inputField"
+        />
+    )
+    
+}
+export default Input;
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/InputField/input.jsx]: add line

File: thirdstage-react/src/component/InputField/inputStyle.scss [hunk 8]
```diff
@@ -0,0 +1,5 @@
+.inputField {
+    border: 2px solid rgb(224, 223, 223);
+    padding: 10px
+
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/InputField/inputStyle.scss]: remove extra line

File: thirdstage-react/src/component/NavBar/navBar.jsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+function NavBar(){
+    return(
+        <section className="navBar">
+            <p className="navBar__logo"><span>BR</span>Archeticts</p>
+            <ul className="navBar__list">
+            <li><a href="#Projects">Projects</a></li>
+            <li><a href="#About">About</a></li>
+            <li><a href="#Contact">Contact</a></li>
+            </ul>
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/NavBar/navBar.jsx]: not formatted well

File: thirdstage-react/src/component/Projects/projects.jsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+function Projects() {
+    return (
+        <div id="Projects" 
+             className="projects">
+            <p className="projects__header">Projects</p>
+            <div className="projects__list">
+                {images.map((img, index) => (
+                    <Card   imgSource={img} 
+                            key={index} />
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/Projects/projects.jsx]: not formatted well

File: thirdstage-react/src/component/cards/AboutCard/aboutCards.jsx [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+                 alt="founders"
+                 className="card__image" />
+            <span className="card__name">john dohe</span>
+            <span className="card__position">Ceo & Founder</span>
+            <p className="card__description">
+                Lorem ipsum dolor sit amet Lorem ipsum dolor,
+                sit amet consectetur adipisicing elit. 
+            </p>
+            <button className="card__button">Contact</button>
+        </div>
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/cards/AboutCard/aboutCards.jsx:15]: - the data here is not static, make multiple data objects in About component and pass it to this component then read it. (same as passing imgSource)
- let's try to use **arrow function** for this component

File: thirdstage-react/src/component/cards/projectsCard/projectsCards.jsx [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+
+
+
+function Card({imgSource}){
+        return(
+            <div className="projects__list__card">
+                <span>summer house</span>
+                <img src={imgSource} 
+                     alt="summer house" />
+            </div>
```
Comment (amonamoqadi) [inline thirdstage-react/src/component/cards/projectsCard/projectsCards.jsx]: check my comment on aboutCard please

File: thirdstage-react/src/pages/MainPage.jsx [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+                <NavBar/>
+                <Header/>
+                <Projects/>
+                <About/>
+                <Contact/>
+        </div>
+        )
+        
+}
+
```
Comment (amonamoqadi) [inline thirdstage-react/src/pages/MainPage.jsx]: not formatted well

File: thirdstage-react/src/component/About/about.jsx (hunk 2 above)
Comment (yousef-najeh) [reply thirdstage-react/src/component/About/about.jsx]: i searched about the best declaration for it and that what i got 
If the array doesn't need to be recreated every time the component renders, declaring it outside the component can improve performance. Constantly recreating the array inside the component can lead to unnecessary work.
and in this case its static

**Pull request closed without merging**